src/components/article-card.jsx

```diff
@@ -0,0 +1,40 @@
+import {useState, useEffect} from 'react';
+import '../test.css';
+import {getArticleByArticleId} from '../utils/api'
+const ArticleCard = (article) => {
+    const postedAt = article.created_at.slice(0, article.created_at.length -5).replace('T', '  ')
+    const topic = `${article.topic.slice(0,1).toUpperCase()}${article.topic.slice(1, article.topic.length)}`
+    const [articleBody, setArticleBody] = useState('');
+    useEffect(() => {
+        if (articleBody === '') {
```

Comment:
This if statement is unnecessary as at the moment the useEffect is only supposed to run once and therefore atm this condition will always evaluate to true